apps/nextjs/src/components/icons/picker/icon-picker.tsx

```diff
@@ -35,33 +35,48 @@ export const IconPicker = ({ initialValue, onChange, error, onFocus, onBlur }: I
 
   const groups = notNullableData.map((group) => {
```

Comment:
You could try if the `useSuspenseQuery` also works for this one, then we wouldn't need to fallback to an empty array